src/hooks/useWalletSectionsData.ts

```diff
@@ -40,6 +42,27 @@ export default function useWalletSectionsData({
   const { data: positions } = usePositions({ address: accountAddress, currency: nativeCurrency });
   const { data: claimables } = useClaimables({ address: accountAddress, currency: nativeCurrency });
 
+  // claimables analytics
+  useEffect(() => {
+    let totalUSDValue = 0;
+    const claimablesUSDValues: {
+      [key: string]: number;
+    } = {};
+
+    claimables?.forEach(claimable => {
+      const attribute = `${claimable.analyticsId}USDValue`;
+      totalUSDValue += claimable.value.usd;
+
+      if (claimablesUSDValues[attribute] !== undefined) {
+        claimablesUSDValues[attribute] += claimable.value.usd;
+      } else {
+        claimablesUSDValues[attribute] = claimable.value.usd;
+      }
+    });
+
+    analyticsV2.identify({ claimablesAmount: claimables?.length ?? 0, claimablesUSDValue: totalUSDValue, ...claimablesUSDValues });
+  }, [claimables]);
```

Comment:
are analytics disabled in development? I'm assuming so, but just verifying so we don't slam this when HMR kicks in.

Another note, maybe worth debouncing this? Do we want to send it every single time the wallet data is used?

Reply:
kinda, analytics are only disabled if `IS_TESTING` env var is true. i think debouncing is a good idea

Reply:
I'd track only if claimables amount > 0, otherwise if data isn't ready we'd track 2 events for the same user, one with 0 and 1 with the remaining amount.

Reply:
i added a throttle to limit this to once every 2 min since that's the refetchInterval for claimables